packages/react-scripts/template/README.md

```diff
@@ -2233,6 +2234,16 @@ Now offers a zero-configuration single-command deployment. You can use `now` to
 
 Details are available in [this article.](https://zeit.co/blog/unlimited-static)
 
+### [Roast](https://www.roast.io/)
+
+[Roast.io](https://www.roast.io/) offers zero-configuration server side rendered (via `ReactDOMServer.renderToString`) hosting on a CDN.
+
+1. Install the `roast` command-line tool via node with `npm install -g roast`
+2. Run `roast deploy` from your project directory
+3. Choose `build` as the path to deploy.
+
+push state URLs and server-side rendering are enabled by default.
```

Comment:
I don't understand this. CRA doesn't support server rendering. Why is this relevant here?